build.gradle

```diff
@@ -1,138 +1,141 @@
+    compile fileTree(dir: 'lib', include: '*.jar')
 }
 
 configurations.all {
-	resolutionStrategy.dependencySubstitution {
-		substitute module('org.bukkit:bukkit') with module('com.destroystokyo.paper:paper-api:1.11.2-R0.1-SNAPSHOT')
-	}
+    resolutionStrategy.dependencySubstitution {
+        substitute module('org.bukkit:bukkit') with module('com.destroystokyo.paper:paper-api:1.11.2-R0.1-SNAPSHOT')
+    }
 }
 
 processResources {
-	filter ReplaceTokens, tokens: [
-		"version" :System.getenv('SKRIPT_VERSION') == null ? project.property("version") : System.getenv('SKRIPT_VERSION')
-	]
+    filter ReplaceTokens, tokens: [
+            "version": System.getenv('SKRIPT_VERSION') == null ? project.property("version") : System.getenv('SKRIPT_VERSION')
+    ]
 }
 
 jar {
-	archiveName System.getenv('SKRIPT_JAR_NAME') == null ? 'Skript.jar' : System.getenv("SKRIPT_JAR_NAME")
-	
-	manifest {
-		attributes("Name": "ch/njol/skript",
-				"Sealed": "true")
-	}
+    archiveName System.getenv('SKRIPT_JAR_NAME') == null ? 'Skript.jar' : System.getenv("SKRIPT_JAR_NAME")
+
+    manifest {
+        attributes("Name": "ch/njol/skript",
+                "Sealed": "true")
+    }
 }
 
 license {
     header file('licenseheader.txt')
+    mapping {
+        java = 'SLASHSTAR_STYLE'
+    }
     exclude('**/Metrics.java')
 }
 
 configurations {
-	ecj
+    ecj
 }
 
 dependencies {
-	ecj 'org.eclipse.jdt.core.compiler:ecj:4.6.1'
+    ecj 'org.eclipse.jdt.core.compiler:ecj:4.6.1'
 }
 
 compileJava {
-	options.fork = true
-	options.encoding = 'UTF-8'
-	options.compilerArgs = ['-properties', '.settings/org.eclipse.jdt.core.prefs', '-encoding', 'UTF-8']
-	options.forkOptions.with {
-		executable = 'java'
-		jvmArgs = ['-classpath', project.configurations.ecj.asPath, 'org.eclipse.jdt.internal.compiler.batch.Main']
-	}
+    options.fork = true
+    options.encoding = 'UTF-8'
+    options.compilerArgs = ['-properties', '.settings/org.eclipse.jdt.core.prefs', '-encoding', 'UTF-8']
+    options.forkOptions.with {
+        executable = 'java'
+        jvmArgs = ['-classpath', project.configurations.ecj.asPath, 'org.eclipse.jdt.internal.compiler.batch.Main']
+    }
 }
 
 task sourceJar(type: Jar) {
     from sourceSets.main.allJava
 }
 
 publishing {
-	publications {
-		mavenJava(MavenPublication) {
-			groupId "ch.njol"
-			artifactId "skript"
-			version project.property("version")
-
-			from components.java
-
-			artifact sourceJar {
-    			classifier "sources"
-			}
-		}
-	}
-	
-	repositories {
-		maven {
-			url "file:///itemapi_repo/"
-		}
-	}
+    publications {
+        mavenJava(MavenPublication) {
+            groupId "ch.njol"
+            artifactId "skript"
+            version project.property("version")
+
+            from components.java
+
+            artifact sourceJar {
+                classifier "sources"
+            }
+        }
+    }
+
+    repositories {
+        maven {
+            url "file:///itemapi_repo/"
+        }
+    }
```

Comment:
Skript's build files use tabs, and that is intentional.

Reply:
Fine, gonna rollback, IntelliJ changed that for me

Reply:
"That is intentional" any reason for that actually?